priv/perf/apps/load_test/lib/common/byzantine_events.ex

```diff
@@ -216,44 +244,63 @@ defmodule OMG.Performance.ByzantineEvents do
   defp map_contract_transaction(enumberable, transaction_function) do
     enumberable
     |> Enum.map(transaction_function)
-    |> Task.async_stream(&Support.DevHelper.transact_sync!(&1, timeout: :infinity),
+    |> Task.async_stream(&Ethereum.transact_sync(&1, 200_000),
       timeout: :infinity,
       max_concurrency: 10_000
     )
     |> Enum.map(fn {:ok, result} -> result end)
     |> List.last()
   end
 
-  # This function is prepared to be called in `WaitFor.ok`.
+  # This function is prepared to be called in `Sync`.
   # It repeatedly ask for Watcher's `/status.get` until Watcher consume mined block
-  defp watcher_synchronized?(root_chain_height, watcher_url) do
-    {:ok, status} = WatcherClient.get_status(watcher_url)
+  defp watcher_synchronized?(root_chain_height, service) do
+    {:ok, status_response} =
+      WatcherSecurityCriticalAPI.Api.Status.status_get(LoadTest.Connection.WatcherSecurity.client())
+
+    status = Jason.decode!(status_response.body)["data"]
 
     with true <- watcher_synchronized_to_mined_block?(status),
-         true <- root_chain_synced?(root_chain_height, status) do
+         true <- root_chain_synced?(root_chain_height, status, service) do
       :ok
     else
       _ -> :repeat
     end
   end
 
-  defp root_chain_synced?(nil, _), do: true
+  defp root_chain_synced?(nil, _, _), do: true
 
-  defp root_chain_synced?(root_chain_height, status) do
+  defp root_chain_synced?(root_chain_height, status, nil) do
     status
-    |> Access.get(:services_synced_heights)
+    |> Map.get("services_synced_heights")
+    |> Enum.reject(fn height ->
+      service = height["service"]
+      # these service heights are stuck on circle ci, but they work fine locally
+      # I think ci machin is not powerful enough
+      service == "block_getter" || service == "exit_finalizer" || service == "ife_exit_finalizer"
```

Comment:
Open an issue for this? :pray:

Reply:
https://github.com/omgnetwork/elixir-omg/issues/1711